UPGRADE-v3.md

```diff
@@ -76,3 +76,10 @@ Rule | Option | Old value | New value
 Rule set | Note
 -------- | ----
 `@PHP56Migration` | was empty
+
+Changes to FixerFactory
+-----------------------
+
+The FixerFactory will now configure configurable fixers with the default config value (`[]`) when the rule
+set configuration for such fixer is set to `null`. Previously the fixers method `::configure` would not be called
```

Comment:
> Previously the fixers method `::configure` would not be called

Previously it was called in https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.1/src/AbstractFixer.php#L55.

Reply:
check, I was referring to https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.16/src/FixerFactory.php#L178